tests/test_views.py

```diff
@@ -4583,5 +4529,269 @@ def test_confirm_user_signal_called_when_user_confirms_email(self):
         assert_equal(mock_signals.signals_sent(), set([auth.signals.user_confirmed]))
 
 
+class TestDraftRegistrationViews(OsfTestCase):
+
+    def setUp(self, *args, **kwargs):
+        super(TestDraftRegistrationViews, self).setUp(*args, **kwargs)
+
+        self.user = AuthUserFactory()
+        self.auth = self.user.auth
+        self.node = ProjectFactory(creator=self.user)
+
+        MetaSchema.remove()
+        ensure_schemas()
+        self.meta_schema = MetaSchema.find_one(
+            Q('name', 'eq', 'Open-Ended Registration') &
+            Q('schema_version', 'eq', 1)
+        )
+        self.draft = DraftRegistrationFactory(
+            initiator=self.user,
+            branched_from=self.node,
+            registration_schema=self.meta_schema,
+            registration_metadata={
+                'summary': {'value': 'Some airy'}
+            }
+        )
+        self.draft.save()
+
+        current_month = datetime.datetime.now().strftime("%B")
+        current_year = datetime.datetime.now().strftime("%Y")
+
+        valid_date = datetime.datetime.now() + datetime.timedelta(days=180)
+        self.valid_embargo_payload = json.dumps({
```

Comment:
These tests were moved from the old registration views tests and were modified in the minimal way to support the changes. 

The testing logic is not new code, and is IMO outside of the scope of this PR.